pkg/flowcli/services/accounts.go

```diff
@@ -214,6 +215,15 @@ func (a *Accounts) Create(
 		return nil, fmt.Errorf("missing configuration, initialize it: flow init")
 	}
 
+	// if more than one key is provided and at least one weight is specified, make sure there isn't a missmatch
+	if len(keys) > 1 && len(keyWeights) > 0 && len(keys) != len(keyWeights) {
```

Comment:
Nice 👍 

Should we consider doing the same for `sig-algo` and `hash-algo` in the future? Currently those flags apply to all of the specified keys